components/badge/stories/badge.stories.js

```diff
@@ -104,7 +107,12 @@ NonSemanticVariants.tags = ["!dev"];
 NonSemanticVariants.parameters = {
 	chromatic: { disableSnapshot: true },
 };
+NonSemanticVariants.storyName = "Non-semantic";
 
+/**
+ * Fixed positioning impacts the border radius of the badge component. The border radius is 0 along the fixed edge of the component. The actual component position is not represented on this page.
+ *
```

Comment:
_Tiny_ nit (feel free to ignore) — should we remove line 114 so there's no trailing line to match the other doc blocks? ✨